Hackney.Shared.HousingSearch/Domain/Contract/Contract.cs

```diff
@@ -1,34 +1,48 @@
-﻿using Hackney.Shared.HousingSearch.Domain.Contract;
-using Hackney.Shared.HousingSearch.Gateways.Models.Contract;
+﻿using Hackney.Shared.HousingSearch.Gateways.Models.Contract;
 using System;
 using System.Collections.Generic;
 
 namespace Hackney.Shared.HousingSearch.Domain.Contract
 {
     public class Contract
     {
-        public static Contract Create(string id, string targetId, string targetType, IEnumerable<QueryableCharges> charges)
```

Comment:
I think this `.Create` pattern could be refactored out here as well and auto-implemented properties used instead since there's no additonal logic in setting the properties.

Reply:
It looks like this is not used after the previous changes were made. Removed.